packages/astro/CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # astro
 
+## 1.0.0
+
+> Astro v1.0 is out! Read the [official announcement post](https://astro.build/blog/astro-1/).
+
+> **Note**: If you need help migrating an existing Astro project to the new Astro v1.0, check out our updated [Migration Guide](https://docs.astro.build/en/migrate/) and [full documentation website](https://docs.astro.build/).
```

Comment:
```suggestion
> **Note**
> If you need help migrating an existing Astro project to the new Astro v1.0, check out our updated [Migration Guide](https://docs.astro.build/en/migrate/) and [full documentation website](https://docs.astro.build/).
```
I think you should be able to do this for a formatted "GitHub Note" now, if you want!